packages/fiori-generator-shared/src/cap/index.ts

```diff
@@ -0,0 +1,77 @@
+import { existsSync, readFileSync } from 'fs';
+import { join } from 'path';
+import type { CapCustomPaths } from '@sap-ux/project-access';
+
+/**
+ * Lightweight synchronous function to get the CAP project custom paths.
+ * A more robust async alternative is `getCapCustomPaths` in `@sap-ux/project-access`.
+ *
+ * @param capProjectPath - path to project root
+ * @returns cap custom paths
+ */
+export function getCapFolderPaths(capProjectPath: string): CapCustomPaths {
+    const capPaths: CapCustomPaths = {
+        app: 'app/',
+        db: 'db/',
+        srv: 'srv/'
+    };
+
+    const cdsrcPath = join(`${capProjectPath}/.cdsrc.json`);
+    const packageJsonPath = join(`${capProjectPath}/package.json`);
+
+    try {
+        for (const folder of Object.keys(capPaths)) {
+            if (!existsSync(join(`${capProjectPath}/${folder}`))) {
```

Comment:
Don't expect we should check for folder existence here. Just continue merging configs in order.